scripts/toggle.sh

```diff
@@ -31,10 +31,14 @@ while getopts :-:BCEb:c:d:e:h:s:S:t:T:w:x:y: OPT; do
 
 			OPTION:
 
-			  --name <name>     Popup name [Default: "default"]
-			  -[BCE]            Flags passed to display-popup
-			  -[bcdehsStTwxy] <value>
-			                    Options passed to display-popup
+			  --name <name>                   Popup name. [Default: "$DEFAULT_NAME"]
+			  --socket-name <value>           Socket name. [Default: "$DEFAULT_SOCKET_NAME"]
+			  --id-format <value>             Popup ID format. [Default: "$DEFAULT_ID_FORMAT"]
+			  --on-init <hook>                Command to run on popup initialization. [Default: "$DEFAULT_ON_INIT"]
+				--popup-before-open <hook>      Hook to run before opening the popup.
```

Comment:
Here is a nearly invisible formatting issue: this line `--popup-before-open` is preceded by a tab, while the other lines are preceded by two spaces.

Also, this argument should be named `--before-open` to maintain consistency with the others.